pilot/pkg/trustbundle/trustbundle.go

```diff
@@ -38,21 +40,31 @@ type TrustAnchorUpdate struct {
 }
 
 type TrustBundle struct {
-	sourceConfig map[Source]TrustAnchorConfig
-	mutex        sync.RWMutex
-	mergedCerts  []string
-	updatecb     func()
+	sourceConfig       map[Source]TrustAnchorConfig
+	mutex              sync.RWMutex
+	mergedCerts        []string
+	updatecb           func()
+	endpointMutex      sync.RWMutex
+	endpoints          []string
+	endpointUpdateChan chan struct{}
+	remoteCaCertPool   *x509.CertPool
 }
 
-var trustBundleLog = log.RegisterScope("trustBundle", "Workload mTLS trust bundle logs", 0)
+var (
+	trustBundleLog               = log.RegisterScope("trustBundle", "Workload mTLS trust bundle logs", 0)
+	remoteTimeout  time.Duration = 10 * time.Second
+)
 
 const (
 	SourceIstioCA Source = iota
 	SourceMeshConfig
 	SourceIstioRA
+	sourceSpiffeEndpoints
+
+	RemoteDefaultPollPeriod = 30 * time.Minute
```

Comment:
Am I right that if during istiod startup URL will be unavailable, the next retry will happen in 30mins?